### PR TITLE
Reduce page cache pressure during I/O heavy operations

### PR DESCRIPTION
**Pull request by petrosagg**
This also brings in the fadvise-enabled docker daemon

Connected to resin-os/resinos#162

### CODE REVIEW

**Comment by petrosagg** [comment]
I just realised that this PR has to wait for a proper release of meta-resin that will include resin-os/meta-resin#499

**Comment by shaunmulligan** [comment]
@petrosagg would these changes also help on other boards like the RPI3? I have see some of these devices also go into RO, its not as often, but does happen

**Comment by petrosagg** [comment]
@agherzan agreed. fixed both issues

**Comment by petrosagg** [comment]
@shaunmulligan now, we haven't observed these kind of memory problems on other boards. However all boards will inherit the fadvise enabled docker when we update meta-resin in them

File: layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/files/fix-mmc-bbb.conf [hunk 1]
```diff
@@ -1 +1,3 @@
-vm.vfs_cache_pressure=400
+vm.min_free_kbytes=8192
+vm.dirty_ratio=5
+vm.dirty_background_ratio=10
```
Comment (floion) [inline layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/files/fix-mmc-bbb.conf]: I think **vm.dirty_background_ratio** already defaults to 10

Comment (petrosagg) [inline layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/files/fix-mmc-bbb.conf]: The defaults are `vm.dirty_ratio=10` and `vm.dirty_background_ratio=20`. I halved them

Comment (floion) [inline layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/files/fix-mmc-bbb.conf]: Hm, okay. I had checked on a bbb and dirty_backgound_ratio was 10 there

**Comment by agherzan** [comment]
@petrosagg we will need a 1.X version of this PR too. As soon as we have that as well we can go ahead and merge these. I would like to avoid merging them separately to keep them tracked and synced.

**Pull request merged**